docs/book/src/appendix/keywords.md

```diff
@@ -0,0 +1,89 @@
+# [Appendix A: Keywords](#appendix-a-keywords)
+
+The following list contains keywords that are reserved for current or
+future use by the Sway language. As such, they cannot be used as
+identifiers. Identifiers are names of functions, variables,
+parameters, modules, constants, attributes, types or
+traits, ect.
+
+## [Keywords Currently in Use](#keywords-currently-in-use)
+
+The following is a list of keywords currently in use, with their
+functionality described.
+
+- `as` - perform primitive casting, or rename items in `use` statements
+- [`abi`](https://fuellabs.github.io/sway/master/book/sway-program-types/smart_contracts.html#the-abi-declaration) - defines a smart contract ABI in a syntactcally similar way to traits
+- [`break`](https://fuellabs.github.io/sway/v0.44.0/book/basics/control_flow.html#break-and-continue) - exit a loop immediately
+- [`const`](https://fuellabs.github.io/sway/v0.44.0/book/basics/constants.html) - define constant items
+- [`continue`](https://fuellabs.github.io/sway/v0.44.0/book/basics/control_flow.html#break-and-continue) - continue to the next loop iteration
+- `dep` - call dependencies outside of the current `Forc.toml` scope
+- `else` - used in conjunction with `if` conditions for control flow constructs
+- [`enum`](https://fuellabs.github.io/sway/v0.44.0/book/basics/structs_tuples_and_enums.html#enums) - define an enumeration
+- `false` - Boolean false literal
+- [`fn`](https://fuellabs.github.io/sway/master/book/basics/functions.html)- define a function or the function pointer type
+- [`if`](https://fuellabs.github.io/sway/v0.44.0/book/basics/control_flow.html#if-expressions) - branch based on the result of a conditional expression
+- `impl` - implement inherent or trait functionality
+- `let` - bind a variable
+- [`match`](https://fuellabs.github.io/sway/v0.44.0/book/basics/control_flow.html#match-expressions) - exhaustfully match a value to patterns
+- `mod` - define a module
+- `mut` - denote mutability in references, or pattern bindings
+- `pub` - denote public visibility of Sway data structures, traits, or modules
+- `ref` - bind by reference
+- `return` - return early from a function
+- `Self` - a type alias for the type we are defining or implementing
+- `self` - method subject
+- [`struct`](https://fuellabs.github.io/sway/v0.44.0/book/basics/structs_tuples_and_enums.html#structs) - define a structure
+- [`trait`](https://fuellabs.github.io/sway/master/book/advanced/traits.html#declaring-a-trait) - define a trait
+- `true` - Boolean true literal
+- `use` - bring symbols into scope
+- `where` - specifies traits for generic types
+- [`while`](https://fuellabs.github.io/sway/v0.44.0/book/basics/control_flow.html#while) - loop conditionally based on the result of an expression
+
+## [Keywords Reserved for Possible Future Use](#keywords-reserved-for-possible-future-use)
```

Comment:
Have you checked that these already cannot be used or are you adding these in apprehension of future language expansion?

If the latter then remove this section.